machine/accton/accton_as_28t_ops/u-boot/platform-as_28t_ops.patch

```diff
@@ -0,0 +1,2795 @@
+Support for Accton AS_28T_OPS
+
+diff --git a/Makefile b/Makefile
+index 4017993..38bf9cc 100755
+--- a/Makefile
++++ b/Makefile
+@@ -316,7 +316,6 @@ LIBS-y += lib/libfdt/libfdt.o
+ LIBS-y += api/libapi.o
+ LIBS-y += post/libpost.o
+ LIBS-y += test/libtest.o
+-LIBS-y += drivers/d1w/libds1wm.o
+ 
+ ifneq ($(CONFIG_AM33XX)$(CONFIG_OMAP34XX)$(CONFIG_OMAP44XX)$(CONFIG_OMAP54XX),)
+ LIBS-y += $(CPUDIR)/omap-common/libomap-common.o
+diff --git a/arch/arm/cpu/armv7/hurricane2/Makefile b/arch/arm/cpu/armv7/hurricane2/Makefile
+index eb4ddaa..0541fe5 100644
+--- a/arch/arm/cpu/armv7/hurricane2/Makefile
++++ b/arch/arm/cpu/armv7/hurricane2/Makefile
+@@ -21,7 +21,7 @@ include $(TOPDIR)/config.mk
+ 
+ LIB	= $(obj)lib$(SOC).o
+ 
+-SOBJS	+= lowlevel_init.o
++SOBJS	+= lowlevel_init.o soc.o
+ 
+ SRCS	:= $(START:.o=.S) $(SOBJS:.o=.S) $(COBJS:.o=.c)
+ 
+diff --git a/arch/arm/cpu/armv7/hurricane2/soc.c b/arch/arm/cpu/armv7/hurricane2/soc.c
+new file mode 100644
+index 0000000..5f846e0
+--- /dev/null
++++ b/arch/arm/cpu/armv7/hurricane2/soc.c
+@@ -0,0 +1,33 @@
++/*
++ * Copyright (c) 2010 Samsung Electronics.
++ * Minkyu Kang <mk7.kang@samsung.com>
++ *
++ * See file CREDITS for list of people who contributed to this
++ * project.
++ *
++ * This program is free software; you can redistribute it and/or
++ * modify it under the terms of the GNU General Public License as
++ * published by the Free Software Foundation; either version 2 of
++ * the License, or (at your option) any later version.
++ *
++ * This program is distributed in the hope that it will be useful,
++ * but WITHOUT ANY WARRANTY; without even the implied warranty of
++ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
++ * GNU General Public License for more details.
++ *
++ * You should have received a copy of the GNU General Public License
++ * along with this program; if not, write to the Free Software
++ * Foundation, Inc., 59 Temple Place, Suite 330, Boston,
++ * MA 02111-1307 USA
++ */
++
++#include <common.h>
++#include <asm/io.h>
++
++#ifndef CONFIG_SYS_DCACHE_OFF
++void enable_caches(void)
++{
++	/* Enable D-cache. I-cache is already enabled in start.S */
++	dcache_enable();
```

Comment:
Any issues observed without this change. 
Not observing any of these changes in bcm95615x.c in iproc uboot, so would be good to understand.